### DISCUSSION TITLE
Welcome to htmx Discussions!

### DISCUSSION THREAD

**Discussion by 1cg** [bigskysoftware/htmx#260] [General]
We are going to give this a go for non-issue related things.

**Comment by 1cg** [comment]
We tried and failed at a disqus forum.  Let's try this.

**Comment by iocouto** [comment]
Are we going to have a separate discussion forum for hyperscript?...

**Comment by 1cg** [comment]
we should probably keep both hyperscript and htmx discussions here until there is critical mass to split them up

**Reply by guzmanojero** [reply]
> We tried and failed at a disqus forum. Let's try this.

Cool! I hope this becomes popular.
